ui/src/NavigationBar/navigationBar.scss

```diff
@@ -16,16 +18,25 @@
     color: darkgreen;
   }
   &__picture {
-    width: 100px;
-    height: 100px;
-    background-image: url("../../public/images/goblin.webp");
-    background-size: 100px;
+    width: 125px;
+    height: 125px;
+    background-size: 125px;
+    background-image: url("#{$baseurl}goblin.png");
     &:hover {
-      background-image: url("../../public/images/Pilbo_Goober.webp");
+      background-image: url("#{$baseurl}Pilbo_Goober.png");
```

Comment:
what is `#{$baseurl}`

Reply:
Defined on line 1, scss allows us to use variables that we can define, so instead of writing the same long path, we just call it like so